python_modules/libraries/dagster-dbt/dagster_dbt/include/scaffold/project.py.jinja

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+
+from dagster_dbt import DbtProject
+
+{{ dbt_project_name }} = DbtProject(
+    project_dir=Path(__file__).joinpath({{ dbt_project_dir_relative_path_parts | join(', ')}}).resolve(),
+    packaged_project_dir=Path(__file__).joinpath("..", "..", "dbt-project").resolve(),
```

Comment:
[1] its not clear to me that wiring up `--use-experimental-dbt-project` to also do the package data stuff is right. I am not sure how negatively the package data steps will impact users who do not intend for it, but it seems worth avoiding 

I think we want
* `--use-dbt-project-package-data-dir` turns on `DbtProject` and package data
* `--use-experimental-dbt-project` only turns on `DbtProject`

Reply:
Spoke offline. Going to do the following in this PR:

- `“--use-experimental-dbt-project”`: Scaffold using DbtProject
- `“--use-dbt-project-package-data-dir”`: Scaffold using DbtProject with its `packaged_project_dir` argument populated.

In a follow-up PR, I'll try to consolidate these two flags by making `packaged_project_dir` create a symlink during local dev.